mmf/utils/build.py

```diff
@@ -152,14 +152,26 @@ def build_dataloader_and_sampler(
     if not isinstance(dataset_instance, torch.utils.data.IterableDataset):
         other_args = _add_extra_args_for_dataloader(dataset_instance, other_args)
 
+    if getattr(training_config, 'device', 'cuda') == 'xla':
```

Comment:
this keeps coming up, if there's a utils file, you can have a `is_xla()` function.